openmdao/utils/coloring.py

```diff
@@ -78,15 +89,26 @@
 # new partial/semi-total coloring.
 _use_partial_sparsity = True
 
-# If True, ignore use_fixed_coloring if the coloring passed to it is _STD_COLORING_FNAME.
+# If True, ignore use_fixed_coloring if the coloring passed to it is STD_COLORING_FNAME.
 # This is used when the 'openmdao partial_coloring' or 'openmdao total_coloring' commands
 # are running, because the intent there is to generate new coloring files regardless of
 # whether use_fixed_coloring was called.
 _force_dyn_coloring = False
 
 # used as an indicator that we should automatically name coloring file based on class module
-# path or system pathname
-_STD_COLORING_FNAME = object()
+# path or system pathname. U
+
+
+class STD_COLORING_FNAME(object):
```

Comment:
Being pedantic here but it's no longer necessary to inherit from object :)

Reply:
fixed